shared/src/utils/util.ts

```diff
@@ -48,3 +48,16 @@ export function isValidUrl(url: string) {
     return false;
   }
 }
+
+export function isValidSubscriptionProtocol(protocol: string) {
```

Comment:
Please use the `SubscriptionProtocol` type instead of string

Reply:
Good eye.

Reply:
Done